content/scout/exceptions.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Manage vulnerability exceptions
```

Comment:
Tazin was keen on addressing (via documentation) the use case where image producers want to tell their image consumers about false positives. We have nothing specifically in Scout to help with this, but in theory an advantage of attaching VEX statements to the registry on a publicly available image is that image consumers can download and view these documents. Not sure whether we can include this advantage in our docs without losing clarity.

Reply:
Added a sentence about it being possible for consumers to inspect attestations from a registry. Doing the actual inspection is a little bit tedious right now, if it means doing it by the CLI (needs multiple `buildx imagetools inspect` commands, or alternatively `crane`) maybe there will be some UI magic that makes this a better experience, I'll have to check when I can do the e2e test with the dashboard.